cmd/tptctl/cmd/config_aws_cloud_account.go

```diff
@@ -118,10 +118,11 @@ var ConfigAwsCloudAccountCmd = &cobra.Command{
 				map[string]string{},
 			),
 			roleName,
+			*callerIdentity.Account,
```

Comment:
This refers to the `externalAccountId` parameter, which is needed to specify which external account we're allowing access to.